user_service/user_service/settings.py

```diff
@@ -111,7 +111,7 @@
 
     "ALGORITHM": "HS256",
     "SIGNING_KEY": os.environ.get('SECRET_KEY'),
-    "VERIFYING_KEY": "",
+    "VERIFYING_KEY": os.environ.get('SECRET_KEY'),
```

Comment:
How do we get a secret key?

Reply:
We can indicate the secret key in the .env file, I will update the README for this. The secret key can be any random string, it just needs to be the same as the secret key indicated in the user service.